tutorials/token-gating-tutorial.md

```diff
@@ -90,7 +88,7 @@ If you need some help to get user addresses, here are a few options:
 
 TGRs are the rules that the wallet assets will be checked against by the Collab.Land. The API will return boolean `true`/`false` for each rule to indicate if the account address fulfills the defined rules or not.
 
-Please see [Configuring your TGRs](https://collabland.freshdesk.com/support/solutions/articles/70000036689-discord-bot-walkthrough) for rule definition details.
+Please see [Configuring your TGRs](../docs/local-development-setup/configuring-tgr) for rule definition details.
```

Comment:
This broken link is failing the build. This doc also used to refer to the Freshdesk doc, that's why I directly referred to the Freshdesk doc. Since the process of defining a TGR is a little different than CC, you can create another doc for it and then link it.